.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: ci
+on: [push]
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        elixir-version: ['1.9', '1.10']
```

Maybe we should be testing on Elixir v1.11, @Kraigie may have an opinion on this. It'll run in parallel so theoretically no major time difference (and we can create up to 256 jobs :smile:)
```suggestion
        elixir-version: ['1.9', '1.10', '1.11']
```

---

👍